randovania/games/samus_returns/logic_database/Area 4 - West.txt

```diff
@@ -192,7 +192,7 @@ Extra - asset_id: collision_camera_004
           Morph Ball
           Any of the following:
               Grapple Beam
-              Phase Drift and Movement (Advanced) and Can Spiderspark
+              Phase Drift and Combat (Advanced) and Spiderspark (Advanced) and Can Spiderspark
```

Comment:
Was this change to combat intentional?

Reply:
Total accident there. Must have accidentally hit the scroll wheel on that box and I didn't notice. Will fix immediately.